npm/test/basics.js

```diff
@@ -560,9 +560,10 @@ test("debug service loading source without entry point attr fails - web worker",
         let m1 = M(q1);
         return [m1];
     }
-}`
+}`,
+      undefined
     );
-    assert.equal(false, result);
+    assert.notEqual("", result);
```

Comment:
Seems a little fragile. If it returns null or undefined, this will pass then? Maybe verify it is a non-zero length string if that is the expectation.